Week0 - einstein

```diff
@@ -0,0 +1,7 @@
+mass = int(input())
```

Comment:
What happens when user enters a value that is not a number? e.g. the character 'a'